KubeArmor/core/karmorprobedata.go

```diff
@@ -0,0 +1,92 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2022 Authors of KubeArmor
+
+package core
+
+import (
+	"encoding/json"
+	"log"
+	"os"
+	"path/filepath"
+
+	cfg "github.com/kubearmor/KubeArmor/KubeArmor/config"
+)
+
+type KarmorData struct {
+	OSImage             string
+	KernelVersion       string
+	KubeletVersion      string
+	ContainerRuntime    string
+	SupportedLSMs       string
+	KernelHeaderPresent bool
+	HostSecurity        bool
+	ContainerSecurity   bool
+	KubeArmorPosture    KubeArmorPostures
+}
+
+type KubeArmorPostures struct {
+	DefaultFilePosture             string
+	DefaultNetworkPosture          string
+	DefaultCapabilitiesPosture     string
+	HostDefaultFilePosture         string
+	HostDefaultNetworkPosture      string
+	HostDefaultCapabilitiesPosture string
+}
+
+func SetKarmorData(dm *KubeArmorDaemon) KarmorData {
+	var kd KarmorData
+	var postures KubeArmorPostures
+
+	lsm := []byte{}
+	lsmPath := "/sys/kernel/security/lsm"
+
+	if _, err := os.Stat(filepath.Clean(lsmPath)); err == nil {
+		lsm, err = os.ReadFile(lsmPath)
+		if err != nil {
+			log.Println(err)
+		}
+	}
+
+	lsms := string(lsm)
+
+	if cfg.GlobalCfg.Policy && !cfg.GlobalCfg.HostPolicy {
+		kd.ContainerSecurity = true
+	} else if !cfg.GlobalCfg.Policy && cfg.GlobalCfg.HostPolicy {
+		kd.HostSecurity = true
+	} else if cfg.GlobalCfg.Policy && cfg.GlobalCfg.HostPolicy {
+		kd.ContainerSecurity = true
+		kd.HostSecurity = true
+	}
```

Comment:
I think we should not have such logic here.
We have plans to keep the config in the configmap anyways. 
I think we can remove this code.
```suggestion
```

Reply:
@daemon1024 , WDYT?

Reply:
All the KubeArmor daemons in a cluster would have the same config map right?

In case of hybrid node setup, it is possible that one of daemons is working and one may not be. How will we differentiate the status in that case?

Reply:
If the daemon is not working then we should not read the tmp file anyways. The configmap would look same. 

Are you implying that different daemons might have different runtime behaviour. And in that case we show the actual runtime behaviour in the tmp file. Is that what is happening in that code snippet?

Reply:
> Are you implying that different daemons might have different runtime behaviour. And in that case we show the actual runtime behaviour in the tmp file.

Right.

> Is that what is happening in that code snippet?

Not sure, like we are printing all the supported LSM instead of active LSM. So needs some updates..

Reply:
i pushed the update for this, kindly review